src/Sentry/SentryOptions.cs

```diff
@@ -223,6 +223,20 @@ public bool IsGlobalModeEnabled
         /// <seealso href="https://docs.sentry.io/platforms/dotnet/configuration/releases/"/>
         public string? Release { get; set; }
 
+        /// <summary>
+        /// The distribution of the application, associated with the release set in <see cref="Release"/>.
+        /// </summary>
+        /// <example>
+        /// TODO: add examples
```

Comment:
given this will appear in intellisense,  probs better to have an issue to track it and remove the todo?